### ISSUE TITLE
Question: Support for "banned" or unsupported emoji's per OS.

### ISSUE THREAD

**Issue by InfamousVague** [rossmacarthur/emojis#13]
Different systems have more or less emoji based on intentional business-political decisions (like removing a gun emoji etc). Particularly on windows a vast majority of emojis do not render. Is it currently possible to obtain a OS "supported" list of emoji within the crate and if not, would this be something I could help get added if you think it'd be useful? For us it seems more logical than maintaining a list of unsupported emojis outside of the crate.

**Comment by rossmacarthur** [comment]
Hi @InfamousVague, is there an existing list like this or do you imagine it would be maintained in this repository?

**Comment by InfamousVague** [comment]
I'm not sure what the best location to store the list is, the issue we're facing is we use the crate to generate a list of emoji's for an emoji picker extension however not all emoji's in this crate that are returned will be natively supported without the use of additional font libraries (for example, a large majority of the flags are missing on Windows).

We could keep this list of "unsupported" emoji (or maybe build the emoji list itself with OS flags in place to prevent unsupported emoji's getting into the list). I'm not sure the best solution yet but I figured it was worth reaching out to see if we could help potentially improve the crate we're using already instead of trying to roll our own solution on top. 

TL;DR not sure the best solution but if you think a solution fits within this crate happy to help get it added!

**Comment by rossmacarthur** [comment]
I'm going to close this because I think solving this problem is out of scope for this crate.
